src/web/client/extension.ts

```diff
@@ -45,6 +46,14 @@ export function activate(context: vscode.ExtensionContext): void {
                         queryParamsMap.set(pair[0], pair[1]);
                     }
                 }
+
+                if (queryParamsMap.get(SITE_VISIBILITY) === PUBLIC) {
+                    const edit: vscode.MessageItem = { isCloseAffordance: true, title: localize("microsoft-powerapps-portals.webExtension.init.sitevisibility.edit","Edit the site") };
```

Comment:
Please confirm if we need: isCloseAffordance

Reply:
yes this is needed for default close and Esc button behavior